lib/tasks/update_job_preferences_roles.rake

```diff
@@ -0,0 +1,21 @@
+namespace :job_preferences do
+  desc "Replace 'other_teaching_support' with 'other_support' in job_preferences.roles"
+  task update_roles: :environment do
+    job_prefs = JobPreferences.where("'other_teaching_support' = ANY(roles)")
+    updated_count = 0
+
+    puts "#{job_prefs.count} JobPreferences to update"
+
+    job_prefs.find_each do |pref|
+      new_roles = pref.roles.map { |role|
+        role == "other_teaching_support" ? "other_support" : role
+      }.uniq
+
+      pref.update!(roles: new_roles)
```

Comment:
it's probably slightly more idiomatic to use assign_attributes and then save!(touch: false) to prevent altering the timestamp